src/styles/blocks/calendar.scss

```
@@ -0,0 +1,48 @@
+.calendar {
+  display: flex;
+
+  flex-wrap: wrap;
+  width: $calendar-width;
+
+  padding: $calendar-padding;
+  padding-top: $calendar-padding * 3;
+  gap: $calendar-gap;
+
+  @each $day, $index in $week-days {
+    &--start-day_#{$day} &__day:first-child {
```

You can remove &__day, it works fine without it
```suggestion
    &--start-day_#{$day} :first-child {
```